src/Resources/PaymentTypes/PaylaterInstallment.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * This represents the Paylater Installment payment type.
+ *
+ * Copyright (C) 2023 - today Unzer E-Com GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @link  https://docs.unzer.com/
+ *
+ * @package  UnzerSDK\PaymentTypes
+ */
+
+namespace UnzerSDK\Resources\PaymentTypes;
+
+class PaylaterInstallment extends BasePaymentType
+{
+    /** @var string $inquiryId */
+    protected $inquiryId;
+
+    /** @var int */
+    protected $numberOfRates;
+
+    /** @var string $iban */
+    protected $iban;
+
+    /** @var string $country */
+    protected $country;
+
+    /** @var string $holder */
+    protected $holder;
```

Comment:
iban and holder are optional, either both are given or none of them
Is this checked somewhere ?

Reply:
There is no validation on SDK side for that. That would be part of integration when collecting data from customer.